README.md

```diff
@@ -23,11 +23,12 @@ Please see /docs/README.md for more information about SDN providers, plugins, an
 To use this project, simply use vagrant to bring up your environment:
 
 ```
-v1k0d3n@machine:~ $ git clone https://github.com/att-comdev/halcyon-vagrant-kubernetes.git
-v1k0d3n@machine:~ $ cd halcyon-vagrant-kubernetes
-v1k0d3n@machine:~ $ git submodule init
-v1k0d3n@machine:~ $ git submodule update
-v1k0d3n@machine:~ $ vagrant up
+$ git clone https://github.com/att-comdev/halcyon-vagrant-kubernetes.git
+$ cd halcyon-vagrant-kubernetes
+$ git submodule init
+$ git submodule update
+$ (cd halcyon-kubernetes; make clean; make; source venv/bin/activate)
```

Comment:
I agree with you, it does seem silly, but I think that providing the virtual-environment steps in a simple script will provide abstraction away from the submodule steps (which are required to get the environment running), where-as some users may not use the virtual environment for deployments.     Another alternative could be that you could even just add a line above line 30  that says" `Optional: Deploy using a python virtual-environment, requires pip package: virtualenv` then list those commands.  This way, it is clear to the user that building the virtual environment requires that package, and is not a truly required step.